### PR TITLE
Add electro documentation to march_input_device

### PR DESCRIPTION
**Pull request by martijnvandermarel**

### CODE REVIEW

File: doc/march_packages/march_input_device.rst [hunk 1]
```diff
@@ -106,7 +104,8 @@ Start the software
     roslaunch march_launch serial_connection.launch
 
 - Make sure the batteries are charged and in the input device.
-- Press on button, which is located on the electronics holder.
+- Make sure the jumper cap in the electronics holder is placed so the input device is powered by the batteries rather than via a USB cable.
```
Comment (Ishadijcks) [inline doc/march_packages/march_input_device.rst:106]: Is a jumper cap a known term? I'm not sure what it means

File: doc/march_packages/march_input_device.rst [hunk 2]
```diff
@@ -71,25 +71,23 @@ Start the software
...
 
-.. note:: If you want to change the network name and password that the input device uses: Change **ssid** and **password** in the *WirelessConnection.h*
-
 - Type in terminal: :code:`hostname -I` Remember the output, this is the hostname. You have to use this address for the next instructions. From this point we call this value **your_hostname**.
 - Uncomment :code:`#define USE_WIRELESS` in the *main.cpp*.
-- Change :code:`IPAddress server(x, x, x, x)` to :code:`IPAddress server(your_hostname)` in the *WirelessConnection.h*
+- Change :code:`IPAddress server(x, x, x, x)` to :code:`IPAddress server(your_hostname)` in the *WirelessConnection.h*.
 - Upload to input device.
 
+.. note:: If you want to change the network name and password that the input device uses: Change **ssid** and **password** in the *WirelessConnection.h*
```
Comment (Ishadijcks) [inline doc/march_packages/march_input_device.rst]: Link to the actual file

Comment (Ishadijcks) [reply doc/march_packages/march_input_device.rst]: :input-device:`WirelessConnection.h <link/to/WirelessConnection>`.

**Review by Ishadijcks — CHANGES_REQUESTED** [review]

File: doc/march_packages/march_input_device.rst (hunk 2 above)
Comment (martijnvandermarel) [reply doc/march_packages/march_input_device.rst]: Old documentation, I just moved it. Probably better to open a new issue for it.

File: doc/march_packages/march_input_device.rst (hunk 1 above)
Comment (martijnvandermarel) [reply doc/march_packages/march_input_device.rst:106]: Depends on the reader. IMO this page is not the place to explain what it is (same as not explaining what a 'port' is, which is earlier on this page). Google tells you the answer quickly enough. I could make the word link to something (e.g. google images result, a confluence page on the ipd). What do you suggest?

File: doc/march_packages/march_input_device.rst (hunk 2 above)
Comment (Ishadijcks) [reply doc/march_packages/march_input_device.rst]: I've added it

**Review by Ishadijcks — APPROVED** [review]

**Pull request merged**